types/split-file/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

Comment:
```suggestion
            "es6"
```
you've @types/ndoe as dependency, why `dom` lib has been added?

Reply:
Because `console.log` would give errors otherwise

Reply:
I will try commiting the suggestion